### PR TITLE
Add more debug to file upload

### PR DESCRIPTION
**Pull request by diversemix**
#### Background

Helps debug #1310

### CODE REVIEW

File: server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js [hunk 1]
```diff
@@ -9,6 +9,17 @@ const { Manuscript, User } = require('@elifesciences/xpub-model')
 const { File } = require('@elifesciences/xpub-model')
 const { S3Storage } = require('@elifesciences/xpub-controller')
 
+async function debugMsFiles(id, user) {
```
Comment (giorgiosironi) [inline server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js:12]: `ms` can easily be confused with milliseconds in the context of manuscripts, given the recent timeout problems

Comment (diversemix) [reply server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js:12]: Its only temporary

**Review by jyenes — APPROVED** [review]

**Review by Cooryd — APPROVED** [review]

**Comment by diversemix** [comment]
Abort!

File: server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js (hunk 1 above)
Comment (giorgiosironi) [reply server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js:12]: also the captain of the Titanic used to say that

Comment (diversemix) [reply server/xpub-server/entities/manuscript/resolvers/uploadManuscript.js:12]: That's why the PR has :sunk:

**Pull request closed without merging**